frontend/src/components/installmentContainer/index.js

```diff
@@ -0,0 +1,76 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Data,
+  InstallmentValue,
+  ResultsContainer,
+} from './CalculatedResultsContainer';
+import PairOfData from '../pairOfData';
+import { SectionTitle } from '../sectionTitle/SectionTitle';
+
+function CalculatedResultsContainer({ title, type, data, addon, fieldNames }) {
+  const parseFields = (key, value) => {
+    if (key === 'interest_type' || key === 'installment_type') {
+      return {
+        fixed: 'stałe',
+        WIBOR: 'zmienne',
```

Comment:
Try to be consistent in naming. Everything should be lower or upper case (I mean object keys)

Reply:
will do in the future :)